inference_interface.py

```diff
@@ -44,38 +45,43 @@ def concatenate_fits(file_names=[], output_name="file.hdf5"):
     raise NotImplementedError()
 
 
-def template_to_multihist(file_name, hist_name=None):
+def template_to_multihist(file_name, hist_name=None, hist_to_read=Histdd):
     """
-    Function that loads a template into the multihist format
+    Function that loads a template into the Hist1d or Histdd format
     :param file_name: name of template file
-    :param hist_name: name of template, if None, return a dict
-    indexed by histogram names containing the histograms
+    :param hist_name: name of template, if None, return a dict indexed by histogram names containing the histograms
     """
     if not HAVE_MULTIHIST:
         raise NotImplementedError("template_to_multihist requires multihist")
-
-    bins = []
-    bin_names = []
     with h5py.File(file_name, "r") as f:
-        for i, (k, b) in enumerate(sorted(f["bins"].items())):
```

Comment:
The `f["bins"]` are always sorted before reading. I think in the PR, we also need to do this.

Reply:
But since here the keys will always be sorted, I think we should be fine: https://github.com/XENONnT/inference_interface/blob/19780a3db1ac3303c0faf266537700d04eda6691/inference_interface.py#L92